lldb/source/Core/IOHandlerCursesGUI.cpp

```diff
@@ -7574,7 +7574,9 @@ IOHandlerCursesGUI::IOHandlerCursesGUI(Debugger &debugger)
 void IOHandlerCursesGUI::Activate() {
   IOHandler::Activate();
   if (!m_app_ap) {
-    m_app_ap = std::make_unique<Application>(GetInputFILE(), GetOutputFILE());
+    m_app_ap = std::make_unique<Application>(
```

Comment:
nit: while you're here, `m_app_ap` -> `m_app_up`? auto pointers are dead, long live auto pointers!

Reply:
Addressed in 776fa2d731c17d6ba0afad2554ebc89cf5e3e5ef